apps/console/src/features/userstores/constants/user-store-constants.ts

```diff
@@ -127,7 +127,7 @@ export const DISABLED = "Disabled";
  * @constant
  * @type {string}
  */
-export const CONSUMER_USERSTORE = "CUSTOMER-DEFAULT";
+export const CONSUMER_USERSTORE = "DEFAULT";
```

Comment:
Why we are having this constant in Identity apps?

Reply:
It should be removed from here. @DonOmalVindula is working on it.